test/test.make

```diff
@@ -137,7 +137,8 @@ test_e2e: start
 .PHONY: run_tests
 test: run_tests
 RUN_TESTS = TEST_WORK=$(abspath _work) \
-	$(TEST_CMD) $(shell $(GO) list $(TEST_ARGS) | sed -e 's;$(IMPORT_PATH);.;')
+	$(TEST_CMD) $(shell $(GO) list $(TEST_ARGS) | grep -v pmem-device-manager | sed -e 's;$(IMPORT_PATH);.;')
```

Comment:
This is a potential trap: if we ever add unit tests for `pmem-device-manager` and forget to remove this, they won't run. It also prevents us from running them via "make test" when they can run (for example, with "sudo make test").

I'd prefer to let it run and rely on the skip check.

Reply:
> I'd prefer to let it run and rely on the skip check.

In this case, when run using `make tests` there will be no information for the user if the tests really run are skipped. Both the cases it shows:

`ok  	github.com/intel/pmem-csi/pkg/pmem-device-manager	0.008s`

Reply:
I don't agree, but have it your way.